scripts/netContain/build_image.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
This entire script is now much safer to run. It doesn't clutter the Docker daemon with images and it also doesn't rebuild the main netplugin image from scratch every single time. The advantage is that this wastes less time, less bandwidth and it's much safer overall.

The risk of running into build errors is also greatly diminished. I've tested this build script thoroughly.